nativeaot/console.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>$(TestTargetFramework)</TargetFramework>

```

Comment:
This project didn't actually meet the naming conventions for the test runner to patch it.